Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -252,3 +251,66 @@ extension DataPipelineImplementation {
         return attributesPlugin
     }
 }
+
+// To process pending tasks in background queue
+extension DataPipelineImplementation {
+    func processIdentifyFromBGQ(identifier: String, body: [String: Any]?) {
```

Comment:
should we be adding timestamp as well?

Reply:
`identify` does not have a `timestamp` in JSON that we get from the queue.
`{"attributes_json_string":"null","identifier":"xyz@customer.io"}`

All event and screen tracks have `timestamp` and have been added to the methods (Refer [this](https://github.com/customerio/customerio-ios/blob/dc754410218d72d9d8d3f87283889cbb1409f8d0/Sources/DataPipeline/DataPipelineImplementation.swift#L271) and [this](https://github.com/customerio/customerio-ios/blob/dc754410218d72d9d8d3f87283889cbb1409f8d0/Sources/DataPipeline/DataPipelineImplementation.swift#L281))

Reply:
These timestamps are from the metric model, we can have timestamp for events from `TaskMetaData`. I added the comment where i think we can probably get and return that information.

Reply:
That is `createdAt` which I believe might differ in terms of the actual `timestamp`. If that works for us then I can utilise and make updates in all the methods.

Reply:
Updated the code with `timestamp` as available, if not then `createdAt` from `QueueTaskMetadata` !!!